### PR TITLE
Updated README and added LICENSE

### PR DESCRIPTION
**Pull request by i3anaan**

### CODE REVIEW

**Comment by i3anaan** [comment]
Additionally, I have updated the installation instructions on Confluence:
https://oms-project.atlassian.net/wiki/spaces/GENERAL/pages/17235970/Installation

File: README.md [hunk 1]
```diff
@@ -2,26 +2,29 @@
 ## OMS-core
 
 ## Description
-The core repository for the Online Management System (OMS), an open-source project of student association AEGEE-Europe.
+The core service repository for the [Online Management System (OMS)](https://github.com/AEGEE/oms-docker), an open-source project of student association [AEGEE-Europe](http://aegee.org/).
```
Comment (linuxbandit) [inline README.md]: online management system?

File: README.md [hunk 2]
```diff
@@ -2,26 +2,29 @@
...
 
-Handles Authentication, Users, Bodies and Circles.
+OMS-core Handles Authentication, Users, Bodies and Circles.
+OMS-core Is build upon Laravel, a PHP framework.
```
Comment (linuxbandit) [inline README.md]: built*

**Review by linuxbandit — CHANGES_REQUESTED** [review]
It's ok except the 2 inconsistencies/typo

File: README.md (hunk 1 above)
Comment (i3anaan) [reply README.md]: Whoops. Fun fact: this is copied from the README in OMS-Docker, so you missed it there :D

**Pull request merged**